test/orange/orangeinp/ConvexRegion.test.cc

```diff
@@ -760,6 +762,85 @@ TEST_F(GenTrapTest, full2)
     EXPECT_VEC_SOFT_EQ((Real3{inf, 20, 40}), result.exterior.upper());
 }
 
+TEST_F(GenTrapTest, adjacent_twisted)
+{
+    /* Lower polygons:
+     *
+     * x=-1      x=1
+     * +----+----+ y=1
+     * |    |    |
+     * |  L |  R |
+     * |    |    |
+     * |    |    |
+     * +----+----+ y=-1
+     *
+     *
+     * Upper polygons:
+     *    x=-0.5
+     * +--+------+ y=1
+     * |   \     |
+     * |    \  R |
+     * |  L  \   |
+     * |      \  |
+     * +-------+-+ y=-1
```

Comment:
How about saving some vertical space by having the Upper polygon sketched by the right-side of the Lower one?  Hope it always comes out in fixed-size fonts on the resulting docs!

Reply:
Yep that would've been smart.